tests/test_libsmoldyn.py

```diff
@@ -8,48 +8,49 @@
 # Test only the extension module. It has to be imported by the user.
 import smoldyn._smoldyn as CppApi
 
+print(CppApi.__version__)
+
 def test_library():
     """We test the C API here
     """
-    CppApi.setBoundaries(([-50, 50], [-50, 50]))
-    assert CppApi.getDim() == 2
-    CppApi.setRandomSeed(1)
-    print('Bounds', CppApi.getBoundaries())
+    s = CppApi.Simulation([-50, -50], [50, 50], ["r", "r"])
+    #  assert s.getDim() == 2
+    s.setRandomSeed(1)
 
-    CppApi.setPartitions("molperbox", 4);
```

Comment:
All functions which require `simptr` as the first argument has moved to the `Simulation` class.